app/Http/Controllers/SetAzureSsoSettingController.php

```diff
@@ -53,6 +54,12 @@ public function __invoke(SetAzureSsoSettingRequest $request)
 
         $azureSsoSettings->save();
 
+        $panel = Filament::getCurrentPanel();
+
+        $panel->domain('https://google.com');
+
+        $panel->save();
```

Comment:
@ankit-canyon I am not sure what this is here. I think it should probably be removed.

Reply:
Hi @Orrison ,

Yeah my bad, I forget to remove this one.

Thank you.